### PR TITLE
Improve meter breakdown pages

### PR DESCRIPTION
**Pull request by ldodds**
Improves the initial version of the meter breakdown pages:

- Adds a "Recent trend" section heading to match similar pages. This section has the original meter breakdown chart and table, plus a new chart that shows monthly consumption by meter for the last 12 months
- Adds a "Long term trend" section if the school has >1 of data. This shows an annual summary of usage by meter
- Adds an "Additional analysis" section that links to tables in other parts of the analysis that shows meter comparisons
- Adds a co2 column to the meter breakdown table so its consistent with other similar tables
- Adds checks whether there's at least 7 days of data before allowing access to basic analysis.

The pages have also been retitled to "Meter comparisons".

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ gem 'pg'
 gem 'scenic'
 
 # Dashboard analytics
-gem 'energy-sparks_analytics', github: 'Energy-Sparks/energy-sparks_analytics', tag: '5.2.13'
+gem 'energy-sparks_analytics', github: 'Energy-Sparks/energy-sparks_analytics', branch: '4388-improve-meter-breakdown-pages'
```
Comment (ldodds) [inline Gemfile:28]: Need to switch to tagged version before merging.

File: app/controllers/schools/advice/base_costs_controller.rb [hunk 2]
```diff
@@ -72,7 +72,7 @@ def aggregate_meter_mpan_mprn
       end
 
       def aggregate_meter_adapter
-        OpenStruct.new(mpan_mprn: aggregate_meter_mpan_mprn, name_or_mpan_mprn: aggregate_meter_label)
+        OpenStruct.new(mpan_mprn: aggregate_meter_mpan_mprn, name_or_mpan_mprn: aggregate_meter_label, has_readings?: true)
```
Comment (ldodds) [inline app/controllers/schools/advice/base_costs_controller.rb:75]: This is an unrelated change that fixes a bug in the meter breakdown charges on costs page, see similar change for baseload

File: spec/system/schools/advice_pages/meter_breakdown_spec.rb [hunk 3]
```diff
@@ -23,6 +23,21 @@
...
+shared_examples 'the meter breakdown is not yet available' do
+  it 'shows not enough data message' do
+    visit path
+    expect(page).to have_content(I18n.t('advice_pages.not_enough_data.title'))
+  end
+end
+
+shared_examples 'the monthly chart is not available' do
+  it 'does not include the chart' do
+    visit path
```
Comment (urbanwide) [inline spec/system/schools/advice_pages/meter_breakdown_spec.rb:35]: Very small thing but I'd put the `visit path` and the `click_on 'Analysis'` in a before block. Same with the one above with the `visit path`. Appreciate it's a one test shared example though so see why you might have done it.

**Review by urbanwide — APPROVED** [review]
Looks good - just one tiny comment 👍
